packages/effect/src/SchemaAST.ts

```diff
@@ -2075,6 +2075,13 @@ export const getTemplateLiteralCapturingRegExp = (ast: TemplateLiteral): RegExp
   return new RegExp(pattern)
 }
 
+/**
+ * @since 3.?.?
```

Comment:
Need to version this function

Reply:
Since this is a bug fix, let's go with version `3.11.3`.